source/_dashboards/iframe.markdown

```diff
@@ -38,6 +38,11 @@ aspect_ratio:
   description: 'Forces the height of the image to be a ratio of the width. Valid formats: Height percentage value (`23%`) or ratio expressed with colon or "x" separator (`16:9` or `16x9`). For a ratio, the second element can be omitted and will default to "1" (`1.78` equals `1.78:1`).'
   type: string
   default: "50%"
+allow_open_top_navigation:
+  required: false
+  description: 'Allow the user to open iframe content links by opening the default browser in Home Assistant mobile app. It is false by default because it adds allow-top-navigation-by-user-activation on the iframe sandbox attribute which is a less secure. So set it to true if you need it and are confident with the iframe content.'
```

Comment:
```suggestion
  description: 'Allow the user to open iframe content links by opening the default browser in the Home Assistant mobile app. It is false by default because it adds allow-top-navigation-by-user-activation on the iframe sandbox attribute which is less secure. So set it to true if you need it and are confident with the iframe content.'
```